relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/stage/strategy/interactive/InteractiveStrategy.kt

```diff
@@ -133,9 +133,13 @@ class InteractiveSimulationController(val maxTurns: Int, val seed: Int, val load
         }
     }
 
+    // Safari doesn't support lookbehinds for the obvious escape processing,
+    // so use a lookahead instead
     private fun String.splitArgs() = trim()
```

Comment:
Argument parsing should really be reworked altogether at some point, but this is fine for now.